lua/wire/wireshared.lua

```diff
@@ -1201,3 +1201,11 @@ hook.Add("PlayerDisconnected", "WireLib_PlayerDisconnect", function(ply)
     tbl[ply] = nil
   end
 end)
+
+-- Notify --
+
+---@alias WireLib.NotifySeverity
```

Comment:
Why is this here? Why not put it with the functions?

Seems like an unnecessary change to this file

Reply:
It was originally a Lua table and I wanted it shared, then it was just left there when converted to alias.